traffic_ops/app/lib/Utils/Helper.pm

```diff
@@ -132,4 +134,18 @@ sub error {
 	);
 }
 
+sub hash_pass {
+	my $pass = shift;
+	return scrypt_hash($pass, \64, 16384, 8, 1, 64);
+}
+
+sub verify_pass {
```

Comment:
how about a comment here describing what's going on?  I assume it's for a gradual upgrade -- trying both crypt and sha1 for passwords that haven't been upgraded yet..